app/helpers/application_helper.rb

```diff
@@ -437,8 +437,8 @@ def hosts_count(resource_name = controller.resource_name)
   end
 
   def webpack_dev_server
-    return unless Rails.env.development?
-    port = ::Rails.configuration.webpack.dev_server.port
+    return unless Rails.env.development? && Rails.configuration.webpack.dev_server.enabled
```

Comment:
Should the `env.development?` check be removed here? If it simply returns based on `dev_server.enabled` then the environment config is authoritative.